pkg/scheduler/framework/plugins/examples/stateful/stateful.go

```diff
@@ -82,9 +82,9 @@ func (mp *MultipointExample) PreBind(ctx context.Context, state *framework.Cycle
 }
 
 // New initializes a new plugin and returns it.
-func New(config *runtime.Unknown, _ framework.Handle) (framework.Plugin, error) {
+func New(ctx context.Context, config *runtime.Unknown, fh framework.Handle) (framework.Plugin, error) {
```

Comment:
`fh framework.Handle` why this change is needed?

Reply:
I forgot to restore the original content that I changed before😅. Updated.